### PR TITLE
Run integration tests in docker

### PR DESCRIPTION
**Pull request by jmthibault79**
No JIRA, but necessary for Jenkins.  Copied from UI and lightly edited.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by jmthibault79** [comment]
jenkins test

**Comment by jmthibault79** [comment]
Getting network failures here.  Any ideas?

`[info] - should ping *** FAILED ***
[info]   akka.stream.StreamTcpException: Tcp command [Connect(leonardo-fiab.dsde-qa.broadinstitute.org:30443,None,List(),Some(10 seconds),true)] failed
[info] - should create, monitor, and delete a cluster *** FAILED ***
[info]   akka.stream.StreamTcpException: Tcp command [Connect(leonardo-fiab.dsde-qa.broadinstitute.org:30443,None,List(),Some(10 seconds),true)] failed`

**Comment by jmthibault79** [comment]
Note: https://broadinstitute.atlassian.net/browse/GAWB-2811 is currently causing all Leo FIAB tests to fail.

**Comment by jmthibault79** [comment]
jenkins test

**Comment by jacmrob** [comment]
jenkins retest

**Comment by jacmrob** [comment]
jenkins retest

File: automation/Dockerfile-tests [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM hseeberger/scala-sbt
```
[the remark this answers is not in the document]
Comment (zarsky-broad) [reply automation/Dockerfile-tests]: `1science/sbt` was the winner.

**Pull request merged**